cassandra-operator/cmd/operator/main.go

```diff
@@ -85,6 +85,10 @@ func handleArgs(_ *cobra.Command, _ []string) error {
 		return fmt.Errorf("invalid metric-poll-interval, it must be a positive integer")
 	}
 
+	if !isPositive(controllerSyncPeriod) && controllerSyncPeriod != 0 {
+		return fmt.Errorf("invalid controller-sync-period, it must not be a negative integer")
```

Comment:
```suggestion
		return fmt.Errorf("invalid controller-sync-period, it must not be a negative duration")
```

Same change required on line 85.